app/models/document.rb

```diff
@@ -0,0 +1,13 @@
+class Document < ActiveRecord::Base
+	belongs_to :case
+	
+	
+has_attached_file :doc, :path => ":rails_root/app/assets/documents/:basename.:extension"
+   validates :doc,presence: true
```

Comment:
Don't use :abc => "xyz" hash rocket format. Please use latest syntax as per Ruby 2.2.3 or greater version. For example abc: 'XYZ'